modules/DoubleClick/resources/mw.DoubleClick.js

```diff
@@ -224,6 +224,12 @@
 	            google.ima.settings.setVpaidMode(vpaidMode);
             };
 
+            var getDisableCustomPlaybackForIOS10Plus = function() {
+                var _disableCustomPlaybackForIOS10Plus =  !mw.getConfig("EmbedPlayer.WebKitPlaysInline") ? false
```

Comment:
why negate then? and also what is `google.ima.ImaSdkSettings.disableCustomPlaybackForIOS10Plus = true;`? IMA ImaSdkSettings doesn't have `disableCustomPlaybackForIOS10Plus` property.
Please revise your code and check, suggest you check playkit-js-ima for reference.